src/joycfg.c

```diff
@@ -185,9 +185,16 @@ volatile int ext=0,left,key;
 volatile int hi,lo;
 volatile int buffer[100];
 
-void key_handler(void)
+void key_handler(void);
+
+asm("key_handler:"
```

Comment:
Trying to follow the change here. Where does `key_handler_impl` call `key_handler`?